src/middlewared/middlewared/plugins/device_/device_info_linux.py

```diff
@@ -175,26 +215,26 @@ def get_disk_details(self, block_device, disk, lshw_disks):
                     disk['serial'] = disk['ident'] = reg.group(1)
 
         if not disk['model'] and os.path.exists(os.path.join(disk_sys_path, 'device/model')):
-            # For nvme drives, we are unable to retrieve it via lshw
```

Comment:
This block is not required as `lsblk` retrieves the model data automatically for nvme drives